includes/embeds/class-amp-soundcloud-embed.php

```diff
@@ -23,9 +30,8 @@ class AMP_SoundCloud_Embed_Handler extends AMP_Base_Embed_Handler {
 	 * Register embed.
 	 */
 	public function register_embed() {
-		if ( function_exists( 'soundcloud_shortcode' ) ) {
-			// @todo Move this to Jetpack.
-			add_shortcode( 'soundcloud', [ $this, 'shortcode' ] );
+		if ( ! $this->is_amp_shortcode_available_in_jetpack( self::SHORTCODE_TAG ) ) {
```

Comment:
Instead of this method, wouldn't it be the same to just do `! class_exists( 'Jetpack_AMP_Soundcloud_Shortcode' )`?  This code will quickly become dead code that should be removed entirely from the AMP plugin, once a new version of Jetpack has been released, so we don't need to worry too much about keeping this code around too long.

Reply:
Sure, https://github.com/ampproject/amp-wp/pull/3678/commits/b64392bf469255d7573e074a8bc0d0d1f1dc14c8 checks whether the class exists.